app/controllers/api/v1/packages_controller.rb

```diff
@@ -365,6 +365,10 @@ def computer_accessory_attributes
         ]
       end
 
+      def medical_attributes
+        %i[brand country_id seriel_number updated_by_id expiry_date]
```

Comment:
```suggestion
        %i[brand country_id serial_number updated_by_id expiry_date]
```

small typing mistake